From 4765245bf15afab123caed34c6331478525e9269 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Fri, 07 Jun 2019 22:51:28 +0300
Subject: [PATCH] Revert "[SW-4185] habanalabs: gaudi: uboot: uart clk 50MHz"

The PCI PLL DIV2 clock output has been restored to its nominal value, so
restore the U-Boot setting too.

This reverts commit aa7d34c58adeef6b7ad6012655e42bfe486c8540.
---

diff --git a/board/habanalabs/gaudi/uboot_asic.fragment b/board/habanalabs/gaudi/uboot_asic.fragment
index 2b3fc2a..032ac38 100644
--- a/board/habanalabs/gaudi/uboot_asic.fragment
+++ b/board/habanalabs/gaudi/uboot_asic.fragment
@@ -2,7 +2,7 @@
 CONFIG_CONARGS="serial0:115200n8"
 CONFIG_BAUDRATE=115200
 CONFIG_BOOTDELAY=0
-CONFIG_DEBUG_UART_CLOCK=50000000
+CONFIG_DEBUG_UART_CLOCK=166000000
 CONFIG_DEFAULT_DEVICE_TREE="gaudi_asic"
 CONFIG_SYS_I2C_EEPROM_BUS=1
 CONFIG_SYS_EEPROM_SIZE=256
